src/__tests__/Button.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Button from '../components/Button';
```

Comment:
As a whole, across all the files (in tests and in component files), it would be nice if our `import` statements for components could be an absolute path: `import Button from 'components/Button';`. I think that can be achieved by adding (in the `webpack.config.js`) to this: https://github.com/achoe314/uptempo/blob/master/webpack.config.js#L39